server/sonar-web/src/main/js/components/SourceViewer/SourceViewerBase.tsx

```diff
@@ -371,11 +352,12 @@ export default class SourceViewerBase extends React.PureComponent {
       // request one additional line to define `hasSourcesAfter`
       to++;
 
-      return this.props
-        .loadSources(this.props.component, from, to, this.props.branch)
-        .then(sources => resolve(sources), onFailLoadSources);
+      return this.props.loadSources!(this.props.component, from, to, this.props.branch).then(
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Forbidden non null assertion ([typescript:S2966](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS2966))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fstas%2Fpopups&issues=AWHb6qYww6XGHKvPuQTG&open=AWHb6qYww6XGHKvPuQTG)

[ISSUE_KEY]: AWHb6qYww6XGHKvPuQTG